client/my-sites/theme/index.web.js

```diff
@@ -29,8 +29,9 @@ function addNavigationIfLoggedIn( context, next ) {
 
 function setTitleAndSelectSiteIfLoggedIn( context, next ) {
 	const theme = getTheme( context.store.getState(), 'wpcom', context.params.slug );
-	if ( theme ) {
-		const themeName = theme.name;
+	const themeOrg = getTheme( context.store.getState(), 'wporg', context.params.slug );
```

Comment:
At this point, it might be nice to create a selector that can handle both wpcom and wporg themes. Not a blocker though